src/frontend/src/utils/enum-pipes.ts

```diff
@@ -69,3 +68,18 @@ export const ChangeRequestStatusTextPipe: (status: ChangeRequestStatus) => strin
       return 'Open';
   }
 };
+
+export const ReimbursementRequestTypeTextPipe: (type: ReimbursementStatusType) => string = (type) => {
```

Comment:
we already have a pipe: 'displayEnum' in pipes.ts that can be used instead of this new helper method